src/sentry/interfaces/stacktrace.py

```diff
@@ -33,6 +33,12 @@
     [a-f0-9]{40}       # sha1
 )/""", re.X | re.I)
 
+# Java Spring specific anonymous classes.
+# see: http://mydailyjava.blogspot.co.at/2013/11/cglib-missing-manual.html
+_java_enhancer_re = re.compile(r'''
+(\$\$[\w_]+?CGLIB\$\$)[a-fA-F0-9]+
```

Comment:
I guess since we're looking at the source code for how this is generated:

https://github.com/cglib/cglib/blob/master/cglib/src/main/java/net/sf/cglib/core/DefaultNamingPolicy.java#L56

It's specifically `ByCGLIB`